<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IMBISTDlg</class>
 <widget class="QDialog" name="IMBISTDlg">
  <property name="windowModality">
   <enum>Qt::WindowModality::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>516</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>BIST (Imec Probe Diagnostics)</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="verticalSpacing">
    <number>12</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>8</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Slot</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="slotSB">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>31</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Port</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="portSB">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="dockLbl">
       <property name="text">
        <string>Dock</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="dockSB">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="testCB">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy">
        <enum>Qt::FocusPolicy::StrongFocus</enum>
       </property>
       <item>
        <property name="text">
         <string>Run All Tests</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Base Station</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Heartbeat</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SerDes Link</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>I2C/MemMap</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>EEPROM</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Shift Registers</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Parallel Serial Bus</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Signal Path</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Noise Level</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="goBut">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Go</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTextEdit" name="outTE">
     <property name="focusPolicy">
      <enum>Qt::FocusPolicy::NoFocus</enum>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="undoRedoEnabled">
      <bool>false</bool>
     </property>
     <property name="lineWrapMode">
      <enum>QTextEdit::LineWrapMode::NoWrap</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="acceptRichText">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>8</number>
     </property>
     <property name="leftMargin">
      <number>8</number>
     </property>
     <property name="rightMargin">
      <number>8</number>
     </property>
     <item>
      <widget class="QPushButton" name="helpBut">
       <property name="text">
        <string>Help</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>60</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="clearBut">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveBut">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>slotSB</tabstop>
  <tabstop>portSB</tabstop>
  <tabstop>dockSB</tabstop>
  <tabstop>testCB</tabstop>
  <tabstop>goBut</tabstop>
  <tabstop>clearBut</tabstop>
  <tabstop>saveBut</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
